jupyter_collaboration/handlers.py

```diff
@@ -184,6 +185,7 @@ async def open(self, room_id):
             try:
                 # Initialize the room
                 await self.room.initialize()
+                self._emit_awareness_event(LogLevel.INFO, self.current_user.name, "join")
```

Comment:
`name` isn't required on the `User` object and it isn't necessarily unique. Let's use `username` here. 

https://github.com/jupyter-server/jupyter_server/blob/df27810d56f4fa501b84dad7284ba0a198e659c8/jupyter_server/auth/identity.py#L40-L44

Reply:
will address